app/lib/connection.rb

```diff
@@ -0,0 +1,25 @@
+module Hammeroids
```

Comment:
There's some style weirdness/inconsistency in here with {} spacing around blocks, also not sure i get the calls in the constructor, but could refactor later?